Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -618,6 +628,56 @@
                         sortable: false,
                         filterable: false
                     },
+                    {
+                        field: "LastChangedByUser.Name",
+                        title: "Sidst redigeret: Bruger",
+                        width: 150,
+                        persistId: "lastchangedname",
+                        template: dataItem => `${dataItem.LastChangedByUser.Name} ${dataItem.LastChangedByUser.LastName}`,
+                        hidden: true,
+                        sortable: true,
+                        filterable: {
+                            cell: {
+                                template: customFilter,
+                                dataSource: [],
+                                showOperators: false,
+                                operator: "contains"
+                            }
+                        }
+                    },
+                    {
+                        field: "LastChanged",
+                        title: "Sidste redigeret: Dato",
+                        format: "{0:dd-MM-yyyy}",
+                        width: 130,
+                        persistId: "lastchangeddate",
+                        template: dataItem => {
+                            // handles null cases
+                            if (!dataItem ||
+                                !dataItem.LastChanged ||
+                                this.moment(dataItem.LastChanged).format(Constants.DateFormat.DanishDateFormat) === "01-01-0001") {
```

Comment:
this complex rendering logic is repeated 4 times in this PR alone, so let's be good citizens, extract the rendering to a helper and reuse it maybe like:

---- As it turns out, such a helper already exists:

RenderFieldsHelper.ts::renderDate

**That however does not take that 01-01-001 case into account, so perhaps extend with that**

Then please update the code you copied this block from
```